kernel/aster-nix/src/syscall/umount.rs

```diff
@@ -0,0 +1,57 @@
+// SPDX-License-Identifier: MPL-2.0
+
+use super::SyscallReturn;
+use crate::{
+    fs::fs_resolver::{FsPath, AT_FDCWD},
+    prelude::*,
+    syscall::constants::MAX_FILENAME_LEN,
+    util::read_cstring_from_user,
+};
+
+pub fn sys_umount(path_addr: Vaddr, flags: u64) -> Result<SyscallReturn> {
```

Comment:
The name could be `sys_umount2` since there is no `umount` syscall signature.

Reply:
I've noticed that the function name of this syscall entry in kernel is `sys_umount` in the latest Linux. See: https://github.com/torvalds/linux/blob/v6.9/include/linux/syscalls.h
So we might keep the function name `sys_umount` here as it is.

However, the syscall name has been changed to `umount2`. See: https://github.com/torvalds/linux/blob/v6.9/arch/x86/entry/syscalls/syscall_64.tbl

Reply:
Thanks for your reminder. So do I need to keep the function name in Xingzhan as sys_umount and change the system call name to umount2?

Reply:
Thanks for your reminder. So do I need to keep the function name in Asterinas as `sys_umount` and change the system call name to `umount2`?